lintOverrides.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  - reason: "This operator uses 1 to 1 and not fan out"
```

Comment:
if there is a way to override this rule to a specific file and/or field, then it'll be good to do that